lib/diagnostic.js

```diff
@@ -14,9 +14,19 @@ db.dropDatabase();
 // House Stark, motto 'Winter is Coming'
 // House Targaryen, motto 'Fire and Blood'
 
-// your code begins here
-
-// your code ends here
+db.houses.insert(
```

Comment:
This should be an array of objects rather than a nested object.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([
  {
    name: 'House Arryn',
    motto: 'As High as Honor'
  },
  {
    name: 'House Stark',
    motto: 'Winter is Coming'
  },
  {
    name: 'House Targaryen',
    motto: 'Fire and Blood'
  }
]);
```